wiki/Client/File_formats/Osz_(file_format)/de.md

```diff
@@ -1,9 +1,3 @@
 # .osz (Dateiformat)
 
-**.osz** ist ein Beatmapdateiformat, welches Hitsounds, .osu Datei, Skin Elemente, .osb Datei und Storyboard Elemente enthalten kann.
-
-Um eine .osz Datei zu erstellen, öffnet man einfach die Beatmap, die man als .osz exportieren möchte, im Editor und klickt oben-links auf "Datei" und dann auf "Paket exportieren", siehe Bild.
-
-![Exporting .osz](img/DE-.osz_export.jpg "Exporting .osz")
-
-Die exportierte Datei wird dann im osu! Verzeichnis im "Export" Ordner gespeichert und aufbewahrt.
+**.osz** ist ein gesamtheitliches Dateiarchiv, welches Beatmaps und alle zusätzlichen Ressourcen wie Skins oder Hitsounds enthält.
```

Comment:
Not sure if this must **explicitely** be just one skin (`wie ein Skin`) or several skins (see current version).

Reply:
`Skinelemente` might be a possibility too 🤔 

You just need to place an image with the correct name in the `.osz` file, and the game uses your custom image when playing the beatmap.

Reply:
yeah, `Skinelemente` sounds best :+1: